include/zephyr/bluetooth/mesh/shell.h

```diff
@@ -38,6 +38,11 @@ struct bt_mesh_shell_target {
 /** @brief External reference to health server */
 extern struct bt_mesh_health_srv bt_mesh_shell_health_srv;
 
+#ifdef CONFIG_BT_MESH_LARGE_COMP_DATA_SRV
```

Comment:
If this `ifdef` needed if `health_srv_meta` is always compiled? And how does `mesh_shell` compiles then without `CONFIG_BT_MESH_LARGE_COMP_DATA_SRV` if it uses `health_srv_meta` unconditionally:
https://github.com/zephyrproject-rtos/zephyr/blob/4fcb831752278b64f78e70b3b9152ee69163f1d0/tests/bluetooth/mesh_shell/src/main.c#L48-L51

Reply:
This is not needed.